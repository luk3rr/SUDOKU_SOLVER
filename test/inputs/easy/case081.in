600800490 000060001 000000702 500090210 060010570 040026900 000400000 010000000 098007643 
