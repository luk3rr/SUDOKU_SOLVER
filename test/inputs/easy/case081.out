Solving the following grid:
6 0 0 | 8 0 0 | 4 9 0 
0 0 0 | 0 6 0 | 0 0 1 
0 0 0 | 0 0 0 | 7 0 2 
------+-------+------
5 0 0 | 0 9 0 | 2 1 0 
0 6 0 | 0 1 0 | 5 7 0 
0 4 0 | 0 2 6 | 9 0 0 
------+-------+------
0 0 0 | 4 0 0 | 0 0 0 
0 1 0 | 0 0 0 | 0 0 0 
0 9 8 | 0 0 7 | 6 4 3 


Solution found :')

6 2 1 | 8 7 3 | 4 9 5 
4 7 9 | 2 6 5 | 3 8 1 
8 3 5 | 9 4 1 | 7 6 2 
------+-------+------
5 8 3 | 7 9 4 | 2 1 6 
9 6 2 | 3 1 8 | 5 7 4 
1 4 7 | 5 2 6 | 9 3 8 
------+-------+------
3 5 6 | 4 8 9 | 1 2 7 
7 1 4 | 6 3 2 | 8 5 9 
2 9 8 | 1 5 7 | 6 4 3 

Algorithm: BFS
Total time: 73 ms
Total expanded states: 12739
